.github/workflows/auto-review-and-pr-summary.yml

```
@@ -0,0 +1,91 @@
+        uses: peter-evans/create-or-update-comment@v3
+        with:
+          token: ${{ secrets.MY_PAT }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: "@coderabbitai review"
+
+      - name: Wait for CodeRabbit summary
+        id: wait_for_comment
+        run: |
+          for i in {1..30}; do
+            sleep 30
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)